monai/data/utils.py

```diff
@@ -28,7 +28,7 @@
 
 def get_random_patch(
     dims: Sequence[int], patch_size, rand_state: Optional[np.random.RandomState] = None
-) -> Tuple[slice, ...]:
+) -> Sequence[slice]:
```

Comment:
in this case the return is for sure a tuple, explicit is better than implicit, so I think the previous type hint is better. If you agree with this, please revise the changes in this PR in a few other places for the same.

Reply:
OK, sounds good.
Thanks.